bin/_mkproject.bat

```diff
@@ -24,8 +24,11 @@ if "%2"=="" goto :missing_name
 if "%3"=="" goto :missing_silo
 
 set PROJECTDIR=%1%2
-set curPROJECT=%1
-set curPROJECTNAME=%2
+:: This is a variable for the function back.bat.
+set CURRENTPROJECT=%1
+set CURRENTPROJECT_NAME=%2
+:: this variable has become obsolete with the addition of MINDBENDER_SILO variable
+:: this can be removed safly if back.bat is removed or changed for something better.
 set PROJECTSTAGE=%3
```

Comment:
Don't forget to remove this.

Use `MINDBENDER_SILO` in scripts where you need this `%3` variable.